src/components/container/SurveyTaskList/SurveyTaskList.stories.tsx

```diff
@@ -56,5 +51,15 @@ CustomStyle.args = {
 		boxShadow:"none"
 	},
 	buttonVariant: "default",
-	buttonColor: "blue"
-}
\ No newline at end of file
+	buttonColor: "blue",
+	sequential: false
```

Comment:
Remove setting sequential to false. Default is false

Reply:
I added so it shows in those stories in the Storybook UI, which is nice to be able to play around with. Is there another way to get it to show if it's the default value? (I'm new to Storybook.)

Reply:
It is part of the Sequential story

Reply:
> It is part of the Sequential story

Yeah, but as I noted in one of my comments, having it available to toggle in *each* story helps test compatibility with all the other settings.  I don't think it's bad to have it available on all of the stories.